src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>$(RepoDefaultTargetFrameworks)</TargetFrameworks>
```

Comment:
Minor - double check and make sure netstandard2.1 can be removed.

Reply:
Updated PR description just now; will use follow-up PRs to decide whether to remove netstandard2.1 or not.

"Introduced a variable "DefaultTargetFrameworks" in `Common.prod.props` with the value: net6.0;netstandard2.0;net462 and align the projects under source folder with the below exceptions:
1. Projects that already target netstandard2.1 will keep nestandard2.1 as one of their targets.
2. netstandard2.0 and ne462 will NOT be added to projects that are only targeting .NET core.
3. Not updating TFM for this OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc as the project is subject to change. https://github.com/open-telemetry/opentelemetry-dotnet/issues/3421#issuecomment-1314630702"